.github/workflows/emscripten.yml

```
@@ -0,0 +1,77 @@
+# Copied from NumPy https://github.com/numpy/numpy/pull/25894
+# https://github.com/numpy/numpy/blob/d2d2c25fa81b47810f5cbd85ea6485eb3a3ffec3/.github/workflows/emscripten.yml
+#
+
+name: Test Emscripten/Pyodide build
+
+on:
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+  # TODO: refine after this is ready to merge
+  push:
+    # branches:
+    #   - main
+    #   - maintenance/**
```

```suggestion
    branches:
      - main
      - maintenance/**
```
It seems like this is close to ready. So we don't forget before merging.

---

Yes, thanks. Let me add this after fixing the failing tests.

---

Addressed in 7b33abd87f56a9ac6e926bff8ac0c74d7e81f79e.